src/main/java/bowlingexercise3/BowlingApplication.java

```diff
@@ -0,0 +1,49 @@
+package bowlingexercise3;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+public class BowlingApplication {
+
+    public static void main(String args[]) throws IOException {
+
+        char press, clear;
+        int i = 0;
+
+        while (i < 10) {
+
+            Scanner scanner = new Scanner(System.in);
+
+            System.out.println("Welcome to 10 Pin Bowling!");
```

Comment:
It's fine to use System.out.println for now - just be aware that we would always use a logger in production code. If you get time, you could investigate how to replace these statements with a logger.